src/django/api/tests/test_email.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework.authtoken.models import Token
+from rest_framework.test import APITestCase
+from django.urls import reverse
+from api.models import (
+    User,
+    Contributor,
+    RequestLog,
+    )
+from django.db import IntegrityError
+
+
+class UserEmailLowerTestCase(APITestCase):
+    def setUp(self):
+        self.email = 'TEST@example.com'
+        self.lower_email = 'test@example.com'
+        self.password = 'password'
+        self.name = 'Test User'
+        self.user = User(email=self.email)
+        self.user.set_password(self.password)
+        self.user.save()
+        Contributor.objects.create(name=self.name, admin=self.user)
+
+        self.accented_email = 'Étienne@example.com'
+        self.lower_accented_email = 'étienne@example.com'
+        self.password_for_accented_email = 'password_two'
+        self.name_for_accented_email = 'Test User Two'
+        self.user_with_accented_email = User(email=self.accented_email)
+        self.user_with_accented_email.set_password(
+            self.password_for_accented_email)
+        self.user_with_accented_email.save()
+        Contributor.objects.create(name=self.name_for_accented_email,
+                                   admin=self.user_with_accented_email)
+
+    def check_email_case(self, **kwargs):
```

Comment:
All the test cases must start with the **test** word. E.g. `test_my_email`

Reply:
The code coverage of the Django app hasn't increased, which means that the test cases were not run.